devops/aws-codebuild/Jenkinsfile.ci

```diff
@@ -1,11 +1,11 @@
 #!groovy
 
```

Comment:
Pipeline has been moved from private server and includes things not appropriate for public one: all jobs should be run on agents instead of the master.